tests/mir-opt/gvn_clone.rs

```diff
@@ -0,0 +1,17 @@
+//@ test-mir-pass: GVN
+//@ compile-flags: -Zmir-enable-passes=+InstSimplify-before-inline
```

Comment:
Is this still useful? Why?

Reply:
I need to eliminate clone shims like `_2 = <u32 as Clone>::clone(move _3)` first.